libmscore/instrument.cpp

```diff
@@ -443,9 +438,13 @@ Channel::~Channel()
 
 void Channel::firePropertyChanged(Channel::Prop prop)
       {
-      QList<ChannelListener *> list(listeners);
+      QList<QPointer<QObject>> list(listeners);
+
+      for (QPointer<QObject> ptr: list) {
+            if (ptr.isNull())
+                  continue;
```

Comment:
This loop seems to do nothing: `continue` statement just switches to the next loop cycle which would anyway be done without this statement.

Reply:
The continue skips over null listeners that are no longer valid.  Hence their propertyChanged() method is not called.

Reply:
Ah, I didn't notice a brace after "for" statement, sorry.